### PR TITLE
Typo, URL and formatting in the build instructions

### PR DESCRIPTION
**Pull request by DelazJ**
The provided URL for the install instructions file was wrong (unless it's broken?)
Add some formatting to text to ease readability
fix some typo and http URL

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -443,7 +446,7 @@ build QGIS.
 
 The free (as in free beer) Community installer is available under:
 
-> http://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe
+https://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe
```
Comment (DelazJ) [inline INSTALL.md:449]: Is it a writing convention to not provide the URL as part of the ongoing paragraph but as a new one?

File: INSTALL.md [hunk 2]
```diff
@@ -375,9 +377,10 @@ new subdirectory called `build` or `build-qt5` in it.
 
 ### 3.11.1. Install build dependencies
 
-```bash
-    dnf install qt5-qtbase-private-devel qt5-qtwebkit-devel qt5-qtlocation-devel qt5-qttools-static qca-qt5-devel qca-qt5-ossl qt5-qt3d-devel python3-qt5-devel python3-qscintilla-qt5-devel qscintilla-qt5-devel python3-qscintilla-devel python3-qscintilla-qt5 clang flex bison geos-devel gdal-devel sqlite-devel libspatialite-devel qt5-qtsvg-devel spatialindex-devel expat-devel proj-devel qwt-qt5-devel gsl-devel postgresql-devel cmake python3-future gdal-python3 python3-psycopg2 python3-PyYAML python3-pygments python3-jinja2 python3-OWSLib qca-qt5-ossl qwt-qt5-devel qtkeychain-qt5-devel qwt-devel sip-devel libzip-devel exiv2-devel
-```
+|Install command for dependencies|
+|--------------------------------|
```
Comment (m-kuhn) [inline INSTALL.md:381]: Does it help if this is a table?

File: INSTALL.md (hunk 1 above)
Comment (m-kuhn) [reply INSTALL.md:449]: I'd actually rewrite to.

Download the [free (as in beer) Community installer](https://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe).

File: INSTALL.md [hunk 3]
```diff
@@ -457,8 +460,8 @@ Download and install following packages:
 | Tool |Website|
 |---------------|---------------|
 |CMake |https://cmake.org/files/v3.12/cmake-3.12.3-win64-x64.msi|
```
Comment (m-kuhn) [inline INSTALL.md:462]: Also, I think it'd be better to replace the table with a list:

 - [CMake](https://cmake.org/files/v3.12/cmake-3.12.3-win64-x64.msi)
 ....

File: INSTALL.md [hunk 4]
```diff
@@ -838,7 +841,8 @@ When configuration is done, click 'OK' to exit the setup utility.
 ### 4.2.10. Compilation and installation
 
 ```cmd
-     make make install
+     make
```
Comment (m-kuhn) [inline INSTALL.md:844]:
```suggestion
```

Not required

File: INSTALL.md (hunk 2 above)
Comment (DelazJ) [reply INSTALL.md:381]: Unlike the code format, my thought is that you don't have to scroll to (try to) read the full list (if ever this is of any interest)

File: INSTALL.md [hunk 5]
```diff
@@ -212,19 +212,21 @@ Now we create a build directory and run ccmake:
...
 When you run ccmake (note the .. is required!), a menu will appear where
-you can configure various aspects of the build. If you want QGIS to have
-debugging capabilities then set CMAKE_BUILD_TYPE to Debug. If you do not have
-root access or do not want to overwrite existing QGIS installs (by your
-package manager for example), set the CMAKE_INSTALL_PREFIX to somewhere you
-have write access to (I usually use ${HOME}/apps). Now press
-'c' to configure, 'e' to dismiss any error messages that may appear.
+you can configure various aspects of the build:
+
+* If you want QGIS to have debugging capabilities then set CMAKE_BUILD_TYPE to Debug.
```
Comment (m-kuhn) [inline INSTALL.md:217]:
```suggestion
* If you want QGIS to have debugging capabilities then set `CMAKE_BUILD_TYPE` to `Debug`.
```

File: INSTALL.md [hunk 6]
```diff
@@ -212,19 +212,21 @@ Now we create a build directory and run ccmake:
...
+* If you do not have root access or do not want to overwrite existing QGIS
+  installs (by your package manager for example), set the CMAKE_INSTALL_PREFIX
+  to somewhere you have write access to (I usually use ${HOME}/apps).
```
Comment (m-kuhn) [inline INSTALL.md:220]:
```suggestion
  to somewhere you have write access to (For example `${HOME}/apps`).
```

File: INSTALL.md [hunk 7]
```diff
@@ -212,19 +212,21 @@ Now we create a build directory and run ccmake:
...
-debugging capabilities then set CMAKE_BUILD_TYPE to Debug. If you do not have
-root access or do not want to overwrite existing QGIS installs (by your
-package manager for example), set the CMAKE_INSTALL_PREFIX to somewhere you
-have write access to (I usually use ${HOME}/apps). Now press
-'c' to configure, 'e' to dismiss any error messages that may appear.
+you can configure various aspects of the build:
+
+* If you want QGIS to have debugging capabilities then set CMAKE_BUILD_TYPE to Debug.
+* If you do not have root access or do not want to overwrite existing QGIS
+  installs (by your package manager for example), set the CMAKE_INSTALL_PREFIX
```
Comment (m-kuhn) [inline INSTALL.md:219]:
```suggestion
  installs (by your package manager for example), set the `CMAKE_INSTALL_PREFIX`
```

**Review by m-kuhn — APPROVED** [review]
Sorry, just realized a couple of things to improve more, ok for you?

**Comment by rduivenvoorde** [comment]
ping @DelazJ

**Comment by DelazJ** [comment]
@rduivenvoorde Afair, they were done as soon as suggested ;) Or did I miss any?

File: INSTALL.md [hunk 8]
```diff
@@ -1028,7 +1027,7 @@ Run from MSYS console in the source directory:
 
 We're going to use precompiled binaries. Use the link below for download:
 
-> http://wwwmaster.postgresql.org/download/mirrors-ftp?file=%2Fbinary%2Fv8.2.4%2Fwin32%2Fpostgresql-8.2.4-1-binaries-no-installer.zip
+http://wwwmaster.postgresql.org/download/mirrors-ftp?file=%2Fbinary%2Fv8.2.4%2Fwin32%2Fpostgresql-8.2.4-1-binaries-no-installer.zip
```
Comment (m-kuhn) [inline INSTALL.md:1030]: This link is broken. And postgres 8.2 is very outdated.
Not sure how we can make this section more future proof and less "maintenance heavy".

Comment (DelazJ) [reply INSTALL.md:1030]: Do we actually need this whole section [4.3 Creation of MSYS environment for compilation of QGIS](https://github.com/qgis/QGIS/blob/master/INSTALL.md#43-creation-of-msys-environment-for-compilation-of-qgis)? We provide an archive of msys at  [4.2.1. MSYS](https://github.com/qgis/QGIS/blob/master/INSTALL.md#421-msys) and knowing that we claim [few lines above (4.2)](https://github.com/qgis/QGIS/blob/master/INSTALL.md#42-building-using-mingw) that building the msys/minGW way is not the preferred road.
IOW do we need to provide ways to do everything from scratch, at the risk of having long and outdated instructions, while we already provide the necessary bricks?

Comment (m-kuhn) [reply INSTALL.md:1030]: I am very open to removing outdated sections (which have been proven that they are not maintained in years)

Comment (jef-n) [reply INSTALL.md:1030]: The msys instructions are unmaintained - and the link to the msys.zip is broken too.

Comment (DelazJ) [reply INSTALL.md:1030]: @jef-n Any pointer to the new link? I looked into https://download.osgeo.org/ with no success; Or do we no longer provide the msys.zip? In which case the only instruction we should keep would be the one suggesting Visual Studio?

**Pull request merged**